app/models/cluster.js

```diff
@@ -1075,7 +1075,22 @@ export default Resource.extend(Grafana, ResourceUsage, {
     return true;
   },
 
-  save(opt) {
+  compareStringArrays(a, b) {
+    let aStr = '';
+    let bStr = '';
+
+    Object.keys(a || {}).forEach((key) => {
+      aStr += `${ key }=${ a[key] },`;
+    });
+
+    Object.keys(b || {}).forEach((key) => {
+      bStr += `${ key }=${ b[key] },`;
+    });
```

Comment:
Doesn't this have to be sorted? I didn't think objects in js guarantee order.

Reply:
Good spot - fixed